src/pages/Product/Product.js

```diff
@@ -0,0 +1,335 @@
+  }, []);
+
+  const onChangeCommentInput = e => {
+    setInputValue(e.target.value);
+    e.target.value.length > 0 ? setOpacity(false) : setOpacity(true);
+  };
+
+  const onSubmitComment = e => {
+    e.preventDefault();
+    //TODO: 백엔드 API 완성 후 구체화 예정
+    // fetch(API, {
+    // method: 'POST',
+    //   headers: {
+    //     Authorization: localStorage.getItem('accessToken'),
+    //   },
+    //  body: JSON.stringify({
+    //   }),
+    // })
+    // .then(res => res.json())
+    // .then(data => )
+  };
+
+  const onClickCommentHeart = () => {
+    //TODO: 백엔드 API 완성 후 구체화 예정
+    // fetch(API, {
+    // method: 'POST',
+    //   headers: {
+    //     Authorization: localStorage.getItem('accessToken'),
+    //   },
+    //  body: JSON.stringify({
+    // })
+    // .then(res => res.json())
+    // .then(data => )
+  };
+
+  const onClickURL = e => {
+    navigator.clipboard.writeText(window.location.href);
+    alert('클립보드에 복사되었습니다.');
+  };
+
+  return (
+    <Wrapper>
+      <Article>
+        <div>
+          <ProductHeader>
+            <div>30평대</div>
+            <div>이틀 전</div>
+          </ProductHeader>
+          <ProductImgContainer>
+            <ProductImg
+              src="https://images.unsplash.com/photo-1501924497965-792fefaea3dc?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1489&q=80"
+              alt="product"
+            />
+          </ProductImgContainer>
+          <div></div>
+        </div>
+        <div>
+          <CommentCount>
+            댓글 <CommentCountNum>{commentList.length}</CommentCountNum>
+          </CommentCount>
+          <InputContainer>
+            <FaSmile className="fas fa-smile"></FaSmile>
+            <form onSubmit={onSubmitComment}>
+              <CommentInput
+                type="text"
+                name="comment"
+                placeholder="칭찬과 격려의 댓글은 작성자에게 큰 힘이 됩니다 :)"
+                value={inputValue}
+                onChange={onChangeCommentInput}
+              />
+              <Register alert={opacity}>등록</Register>
+            </form>
+          </InputContainer>
+          {commentList &&
+            commentList.map(comment => {
+              return (
+                <Comment key={comment.id}>
+                  <Circle alt="user face" src={comment.img_url} />
+                  <CommentFeed>
+                    <UserAndContent>
+                      <User>{comment.user}</User>
+                      <div>{comment.content}</div>
+                    </UserAndContent>
+                    <CommentInfo>
+                      <Time>{comment.date}</Time>
+                      <CommentHeart onClick={onClickCommentHeart}>
+                        <FaHeart className="far fa-heart"></FaHeart>
+                        <span>{comment.like}</span>
+                      </CommentHeart>
+                    </CommentInfo>
+                  </CommentFeed>
+                </Comment>
```

Comment:
```suggestion
          {commentList &&
            commentList.map(({id,img_url,user,content,date,like})=> {
```
인자부분에서 바로 구조분해 할당 할 수도 있습니다!
참고해주세요